test/distributed/fsdp/test_checkpoint_wrapper.py

```diff
@@ -27,7 +27,12 @@ def setUp(self):
 
     def test_load_activation_checkpointed_module(self):
         lin = nn.Linear(10, 10, bias=False)
-        lin = checkpoint_wrapper(lin)
+        lin = checkpoint_wrapper(
+            lin,
+            checkpoint_fn=checkpoint,
+            # checkpoint kwargs
```

Comment:
Is there any case we want to use the `*checkpoint_fn_args` at the moment?

Reply:
Not at the moment.